### PR TITLE
[MIG]13.0 base tier validation

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
functionality tested together with purchase_tier_validation

work done:
- all the usual migration to 13 work (api.multi, 12.0->13.0. remove migration scripts..)
- fix the toggle button
- fix the tests adding cache invalidation. Otherwise the one2many review_ids does not work when needed

### CODE REVIEW

**Comment by pegonzalezspesol** [comment]
@angelmoya please review

File: base_tier_validation/tests/test_tier_validation.py [hunk 1]
```diff
@@ -233,9 +243,63 @@ def test_12_approve_sequence(self):
...
+        )
+        # Request validation
+        review = test_record.with_user(self.test_user_2).request_validation()
+        self.assertTrue(review)
+        self.assertTrue(self.test_user_1.get_reviews({"res_ids": review.ids}))
+        self.assertTrue(self.test_user_1.review_ids)
+        for review in self.test_user_1.review_ids:
+            review.invalidate_cache()
+        self.test_user_1.invalidate_cache()
+        self.test_user_1.review_user_count()
```
Comment (LoisRForgeFlow) [inline base_tier_validation/tests/test_tier_validation.py]: no assert of the count after this?

**Review by LoisRForgeFlow — COMMENTED** [review]
Thanks for the work!

Code review looks great with a minor comment inline.

File: base_tier_validation/tests/test_tier_validation.py (hunk 1 above)
Comment (pegonzalezspesol) [reply base_tier_validation/tests/test_tier_validation.py]: Absolutely! Indeed it's needed more tests to improve the code coverage.. I'm on it and soon will come a new commit with more tests and asserts

**Review by angelmoya — APPROVED** [review]
LGTM :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by LoisRForgeFlow** [comment]
Code looks good and you have tested it, I will functionally test in the purchase tier validation PR. So Let's merge this:

/ocabot merge

thank you!

File: base_tier_validation/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Base Tier Validation",
+    "summary": "Implement a validation process based on tiers.",
+    "version": "13.0.1.0.0",
+    "development_status": "Mature",
+    "maintainers": ["lreficent"],
```
Comment (LoisRForgeFlow) [inline base_tier_validation/__manifest__.py:8]: @OCA-git-bot :point_up:

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**